idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/proofs/ProofsIdePlugin.kt

```diff
@@ -20,24 +22,31 @@ import org.jetbrains.kotlin.psi.KtClassOrObject
 import org.jetbrains.kotlin.psi.KtFunction
 import org.jetbrains.kotlin.psi.KtNamedFunction
 import org.jetbrains.kotlin.psi.KtProperty
+import arrow.meta.invoke as cli
 
 val IdeMetaPlugin.typeProofsIde: IdePlugin
   get() = "Type Proofs IDE" {
     meta(
+      // refinementAnnotator(),
       proofLineMarkers(ArrowIcons.SUBTYPING, KtNamedFunction::isCoercionProof),
       proofLineMarkers(ArrowIcons.INTERSECTION, KtNamedFunction::isExtensionProof),
       proofLineMarkers(ArrowIcons.REFINEMENT, KtClass::isRefinementProof),
       proofLineMarkers(ArrowIcons.ICON1, KtClassOrObject::isGivenProof),
       proofLineMarkers(ArrowIcons.ICON1, KtProperty::isGivenProof),
       proofLineMarkers(ArrowIcons.ICON1, KtFunction::isGivenProof),
       refinementLineMarkers(),
-//      refinementAnnotator(),
       proofsKotlinCache,
-      coerceProofLineMarker(ArrowIcons.ICON4),
       addDiagnosticSuppressorWithCtx { suppressProvenTypeMismatch(it) },
-      //makeExplicitCoercionIntention(ctx),
-      //makeImplicitCoercionIntention(ctx),
+      coercionLineMarker,
+      explicitCoercionInspection,
+      implicitCoercionInspection,
       codeFolding
     )
   }
 
+val IdeMetaPlugin.typeProofsCli: CliPlugin
+  get() = "Type Proofs Cli Integration".cli {
+    meta(
+      suppressDiagnostic { suppressProvenTypeMismatch(it) }
```

Comment:
Not sure why that is added here?

Reply:
shouldn't be? can I remove it?

Reply:
Isn't it needed to ensure the IDE does not flag coercion call sites as errors because types do not match?

Reply:
nop